cl/favorites/tests.py

```diff
@@ -333,6 +334,33 @@ def test_user_can_change_favorites(self) -> None:
         self.assert_text_in_node("Renamed Favorite", "body")
         self.assert_text_in_node("Modified Notes", "body")
 
+    def test_revert_usertag(self):
+        # Can we revert an object that is being tracked with django-pghistory?
+
+        tag_name = "test-tag"
+        params = {"username": "kramirez", "password": "password"}
+        test_user = UserProfileWithParentsFactory.create(
+            user__username=params["username"],
+            user__password=make_password(params["password"]),
+        )
+        test_tag = UserTag.objects.create(
+            user=test_user, name=tag_name, title="Test tag"
+        )
+
+        path = reverse(
+            "view_tag",
+            kwargs={"username": params["username"], "tag_name": tag_name},
+        )
+        request = RequestFactory().get(path)
+
+        # Increment view counter by 1
+        increment_view_count(test_tag, request)
+        self.assertEqual(test_tag.view_count, 1)
+
+        # Revert object to last change, and check view counter
+        test_tag.history.order_by("-pgh_id")[0]
```

Comment:
Huh? Does looking at the history table automatically revert from it? Where did the revert happen??

Reply:
My mistake, i didn't write the whole line in the test